.github/workflows/api-augment-publish.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish api-augment to npm.js
```

Comment:
Why do we need to publish `api-augmet`? I thought with `types-bundle` it was enough for our pjs fork.

Reply:
types bundle only gives RPC and RuntimeAPI methods.

You need apiAugment to show all autoscraped query, consts and extrinsic calls